From: Rod Whitby <rod@whitby.id.au>
Subject: Fix the FSG-3 kernel command line root

In recent FSG-3 kernels, the hard disk is /dev/sda, not /dev/hda.

Signed-off-by: Rod Whitby <rod@whitby.id.au>
--

Index: apex/src/mach-ixp42x/slugos-fsg3-arm_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-fsg3-arm_config	2008-05-28 10:53:07.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-fsg3-arm_config	2008-05-28 10:53:11.000000000 +0930
@@ -147,9 +147,9 @@
 #    Overrides
 #
 CONFIG_ENV_DEFAULT_CMDLINE_OVERRIDE=y
-CONFIG_ENV_DEFAULT_CMDLINE="root=/dev/hda1 rootdelay=10 console=ttyS0,115200"
+CONFIG_ENV_DEFAULT_CMDLINE="root=/dev/sda1 rootdelay=10 console=ttyS0,115200"
 CONFIG_ENV_DEFAULT_CMDLINE_ALT_P=y
-CONFIG_ENV_DEFAULT_CMDLINE_ALT="root=/dev/hda2 rootdelay=10 console=ttyS0,115200"
+CONFIG_ENV_DEFAULT_CMDLINE_ALT="root=/dev/sda2 rootdelay=10 console=ttyS0,115200"
 # CONFIG_ENV_DEFAULT_STARTUP_OVERRIDE is not set
 # CONFIG_ENV_DEFAULT_STARTUP_ALT_P is not set
 CONFIG_USES_NOR_BOOTFLASH=y
Index: apex/src/mach-ixp42x/slugos-fsg3-armeb_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-fsg3-armeb_config	2008-05-28 10:53:07.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-fsg3-armeb_config	2008-05-28 10:53:11.000000000 +0930
@@ -147,9 +147,9 @@
 #    Overrides
 #
 CONFIG_ENV_DEFAULT_CMDLINE_OVERRIDE=y
-CONFIG_ENV_DEFAULT_CMDLINE="root=/dev/hda1 rootdelay=10 console=ttyS0,115200"
+CONFIG_ENV_DEFAULT_CMDLINE="root=/dev/sda1 rootdelay=10 console=ttyS0,115200"
 CONFIG_ENV_DEFAULT_CMDLINE_ALT_P=y
-CONFIG_ENV_DEFAULT_CMDLINE_ALT="root=/dev/hda2 rootdelay=10 console=ttyS0,115200"
+CONFIG_ENV_DEFAULT_CMDLINE_ALT="root=/dev/sda2 rootdelay=10 console=ttyS0,115200"
 # CONFIG_ENV_DEFAULT_STARTUP_OVERRIDE is not set
 # CONFIG_ENV_DEFAULT_STARTUP_ALT_P is not set
 CONFIG_USES_NOR_BOOTFLASH=y
